### DISCUSSION TITLE
What can I do for improve the site ?

### DISCUSSION THREAD

**Discussion by sasiruLK** [markerikson/react-community-tools-practices-cheatsheet#27] [Q&A]
I just found out about the site and I was looking at the menus like starting with react or starting a new project. Can I add content to those or how do I improve them?

**Comment by markerikson** [comment]
Yeah, the main thing that's needed atm is filling out content for sections like "State Management", "Styling", "Build Tools", and "Data Fetching" - see some of the open discussion issues for ideas, and then file PRs to add / improve pages!

**Reply by sasiruLK** [reply]
Awesome, I'll check the issues and discussions.
